### DISCUSSION TITLE
Consider new model types in the ensemble

### DISCUSSION THREAD

**Discussion by robbinscalebj** [robbinscalebj/NeonPredictability#15] [General]
Are there more models to add to the ensemble? For example, more from package 'modeltime', e.g., prophet, arima_xgboost

**Comment by robbinscalebj** [comment]
"As many models as possible" is good, but we want to have diversity in model types and I think not stack the ensemble with a bunch of models that are all very small variations on the same theme. Do we want to have the same driver data for all models that use 'exogenous regressors' (e.g., not ARIMA)?

**Comment by OlssonF** [comment]
I think Prophet might be a good addition. It performed well in aquatics and seems fairly flexible

**Comment by abbylewis** [comment]
Do we need a new prophet model? Or can we just use Carl's? Is he submitting across all variables?

**Comment by abbylewis** [comment]
It could also be interesting to include ensemble forecasts

**Comment by abbylewis** [comment]
Thoughts on how/whether we use persistence and climatology in this analysis?

**Comment by OlssonF** [comment]
> Do we need a new prophet model? Or can we just use Carl's? Is he submitting across all variables?

yeah, he is. So if you are happy with that, there's no point reinventing the wheel.

**Comment by OlssonF** [comment]
> It could also be interesting to include ensemble forecasts

Yes! Love this idea. I have code that I can share for generating ensemble if you want

**Comment by robbinscalebj** [comment]
> > It could also be interesting to include ensemble forecasts
> 
> Yes! Love this idea. I have code that I can share for generating ensemble if you want

@OlssonF That would be great, would you be able to upload that script to the repo soon? Or maybe it's more complex than that. Specifically, can I make an Issue for this and assign it to you?

One thing this brings up: Could it make sense to ONLY include the skill of the proper ensemble forecasts in the GAM analyses? Like, right now, the normalized NSE is calculated by averaging the predictions of each model and each horizon, and comparing that average to the observed target (aggregated by forecast week).

**Comment by robbinscalebj** [comment]
> > Do we need a new prophet model? Or can we just use Carl's? Is he submitting across all variables?
> 
> yeah, he is. So if you are happy with that, there's no point reinventing the wheel.

 Making "add Carl's Prophet model' an Issue assigned to myself.
